code/game/objects/structures/cannons/mounted_guns/mounted_gun.dm

```diff
@@ -0,0 +1,187 @@
+//Mounted guns are basically a smaller equivalent to cannons, designed to use pre-existing ammo rather than cannonballs.
+//Due to using pre-existing ammo, they dont require to be loaded with gunpowder or an equivalent.
+
+/obj/structure/mounted_gun
+	name = "Mounted Gun"
+	desc = "Default mounted gun for inheritance purposes."
+	density = TRUE
+	anchored = FALSE
+	icon = 'icons/obj/weapons/cannons.dmi'
+	icon_state = "falconet_patina"
+	var/icon_state_base = "falconet_patina"
+	var/icon_state_fire = "falconet_patina_fire"
+	max_integrity = 300
+	///whether the cannon can be unwrenched from the ground. Anchorable_cannon equivalent.
+	var/anchorable_gun = TRUE
+	//Max shots per firing of the gun.
+	var/max_shots_per_fire = 1
+	//Shots currently loaded. Should never be more than max_shots_per_fire.
+	var/shots_in_gun = 1
+	//shots added to gun, per piece of ammo loaded.
+	var/shots_per_load = 1
+	//Accepted "ammo" type
```

Comment:
Add a slash to these comments for autodoccing purposes.

Reply:
Like this?

Reply:
Yep